src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/CommandLineParserService.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.IO;
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.VisualStudio.ProjectSystem.LanguageServices
+{
+    [Export(typeof(ICommandLineParserService))]
+    internal class CommandLineParserService : ICommandLineParserService
```

Comment:
Can't we just MEF import the Roslyn implementation of this [service](http://source.roslyn.io/#Microsoft.CodeAnalysis.Workspaces/Workspace/Host/CommandLine/ICommandLineParserService.cs,709388161bb14d46,references) instead of re-implementing it here?